app/templates/play/campaign-view.jade

```diff
@@ -1,165 +1,182 @@
-a(href="/").picoctf-hide
-  img.small-nav-logo(src="/images/pages/base/logo.png", title="CodeCombat - Learn how to code by playing a game", alt="CodeCombat")
+if view.isProduction() && !me.isPremium()
+  // TODO: loading this multiple times yields script error:
+  // Uncaught TagError: adsbygoogle.push() error: All ins elements in the DOM with class=adsbygoogle already have ads in them.
+  .ad-container.picoctf-hide
```

Comment:
`.picoctf-hide` works here, but are we supposed to be populating ads that aren't actually visible? Maybe better to check `window.serverConfig.picoCTF` and use that as a flag to not include the ad script in the first place.